test/server/cards/01_SOR/YodaOldMaster.spec.js

```diff
@@ -0,0 +1,68 @@
+describe('Yoda, Old Master', function() {
+    integration(function() {
+        describe('Yoda\'s When Defeated ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['yoda#old-master'],
+                        hand: ['vanquish'],
+                        deck: ['foundling', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine']
+                    },
+                    player2: {
+                        groundArena: ['vigilant-honor-guards'],
+                        deck: ['foundling', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine']
+                    }
+                });
+            });
+
+            it('should draw a card for each selected player', function () {
+                this.player1.clickCard(this.yoda);
+                this.player1.clickCard(this.vigilantHonorGuards);
+                expect(this.yoda).toBeInLocation('discard');
+                expect(this.player1).toHaveEnabledPromptButtons(['Me', 'Opponent', 'Me and Opponent', 'No one']);
+
+                this.player1.clickPrompt('Me');
+                expect(this.player1.hand.length).toBe(2);
+                expect(this.player2.hand.length).toBe(0);
+                expect(this.player2).toBeActivePlayer();
+            });
+
+            it('should draw a card for each selected player', function () {
+                this.player1.clickCard(this.yoda);
+                this.player1.clickCard(this.vigilantHonorGuards);
+                expect(this.yoda).toBeInLocation('discard');
+                expect(this.player1).toHaveEnabledPromptButtons(['Me', 'Opponent', 'Me and Opponent', 'No one']);
```

Comment:
No need to change, just a note for the future - in a situation where there are multiple tests that set up the exact same scenario, I'm okay if we don't repeat the exact same expectation every time. So e.g. in this case, since the these two expectations are covered in the first test, we could also remove them in subsequent tests just to make the overall code a little shorter.

It's totally personal preference, I'm very fine to keep this as-is if you prefer. Just an fyi in case you want to have to put in less code in the future.